install/installer/cmd/render_test.go

```diff
@@ -75,6 +76,8 @@ func TestRender(t *testing.T) {
 			f, err := os.OpenFile(goldenPath, os.O_RDWR, 0644)
 			defer f.Close()
 			require.NoError(t, err)
+			cmd := exec.Command("sed", "-i", "-e", "\"s/[[:space:]]*$//\"", goldenPath)
+                        defer cmd.Run()
```

Comment:
Finding: The code below,

```go
			// Compare
			require.Equal(t, string(content), got)
```

compares the live-generated, in-memory result with the golden file. So, fixing the golden file in post-processing does not work here.

We'd need to either:
- Save the live-generated, in-memory result to a file, then post-process, then compare with the (also clean) golden file
- Or, clean up trailing spaces in the live-generated, in-memory result (preferable)